ibis/expr/analysis.py

```diff
@@ -826,7 +826,7 @@ def __init__(self, parent, proj_exprs):
         )
         self.clean_exprs = list(map(windowize_function, self.resolved_exprs))
 
-    def get_result(self):
+    def get_result(self, **kwargs):
```

Comment:
Can you elaborate a bit on why this is necessary?

Reply:
It might be a leftover. Not related to UDT; please skip.